script/run-unit-test.js

```diff
@@ -55,7 +55,7 @@ if (options.help) {
 }
 
 const mochaPath = `BABEL_ENV=test NODE_ENV=test mocha ${reporterOption}`;
-const coveragePath = `NODE_ENV=test nyc --all ${coverageInclude} --reporter=lcov --reporter=text --reporter=json-summary mocha --reporter mocha-junit-reporter --no-color --retries 5`;
+const coveragePath = `NODE_ENV=test nyc --all ${coverageInclude} --reporter=json-summary mocha --reporter mocha-multi-reporters --reporter-options configFile=config/mocha-multi-reporter.json --no-color --retries 5`;
```

Comment:
I removed the `lcov` reporter because it doesn't look like we're using it anywhere. The `text` reporter just logs coverage stats, and we already have that outputted in the unit test summary